Makefile

```diff
@@ -46,3 +55,7 @@ package: upgradepip upgradebuild upgradesetuptools
 	$(eval PACKAGE=$(shell python -c "from pep517.meta import load; metadata = load('.'); print(metadata.version)"))
 	tar -zxvf "dist/gridstatus-${PACKAGE}.tar.gz"
 	mv "gridstatus-${PACKAGE}" unpacked
+
+.PHONY: docs
+docs: clean
```

Comment:
i believe this is a command to build the docs from the root of the project. can you update contributing.md to mention this command? 

the 3 item in the section on building the docs

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/371792/209732039-86ddc13c-77cb-42f6-9201-bc93a0a9489c.png">

Reply:
Sure, fixed:
https://github.com/kmax12/gridstatus/pull/71/commits/7331383f56e039ede47bc14db6764fda479527fd